setup.py

```diff
@@ -24,7 +24,7 @@
     ''',
     package_data = { '': ['README.rst'] },
     tests_require=tests_require,
-    install_requires=['rq>=0.3.5'] + tests_require,
+    install_requires=['rq>=0.3.5','croniter==0.3.5'] + tests_require,
```

Comment:
`croniter>0.3.5` doesn't support python 2.6 because they use `timedelta.total_seonds()` in their code. I already created a PR for that: https://github.com/taichino/croniter/pull/45. But I don't know when this will merge.

Why do you find pinning a package dependency to a specific version a bad idea? Although libraries say that they are backwards compatible, most of the times they are not and things suddenly break.
- http://stackoverflow.com/a/28510546/1970722
- http://nvie.com/posts/pin-your-packages/